bower.json

```diff
@@ -11,14 +11,14 @@
     "url": "git://github.com/PolymerElements/iron-overlay-behavior.git"
   },
   "dependencies": {
-    "polymer": "Polymer/polymer#v0.8.0-rc.7",
-    "iron-fit-behavior": "PolymerElements/iron-fit-behavior#^0.8.0"
+    "polymer": "Polymer/polymer#v0.9.0-rc.1",
+    "iron-fit-behavior": "PolymerElements/iron-fit-behavior#^0.9.0"
   },
   "devDependencies": {
-    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.0",
-    "iron-flex-layout": "PolymerElements/iron-flex-layout#^0.8.0",
-    "iron-test-helpers": "PolymerElements/iron-test-helpers#^0.8.0",
-    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.9.0",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^0.9.0",
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#^0.9.0",
+    "test-fixture": "PolymerElements/test-fixture#^0.9.0",
     "web-component-tester": "*",
     "webcomponentsjs": "^0.6.0"
```

Comment:
I think this is supposed to be `webcomponents/webcomponentsjs#^0.6.0`, but I'm not sure that it matters.